### ISSUE TITLE
Improve "File not found" error when running `bun index.html`

### ISSUE THREAD

**Issue by MyRaspberry** [oven-sh/bun#9259]
### What version of Bun is running?

1.0.30 (1424a196)

### What platform is your computer?

Linux 6.1.0-rpi8-rpi-v8 aarch64

### What steps can reproduce the bug?

bun index.html

error: File not found "index.html"

bunx index.html

error: package "index.html" with tag "latest" not found, but package exists

bun run index.html

error: Script not found "index.html"

### What is the expected behavior?

hoping BUN be more beginner friendly

even if that calls are never supposed to work,
the error msg 

> file not found "index.html"

is wrong 

if i call x.html from 
**BUN or BUNX  or BUN RUN**
obviously i want BUN to serve it
why wouldn't you want to do that?

**Comment by Jarred-Sumner** [comment]
thank you for this feedback

i agree with you on all points

**Comment by paperclover** [comment]
a better error for this would be "index.html" cannot be run as a script

and if we want to be very beginner friendly, we can pick up on the `html` extension and link to a documentation page on how to serve files with bun.

**Comment by MyRaspberry** [comment]
? when BUN starts, i assume it collects info about THIS path content? like look for a package.json ....

? could it be in a situation ( after my 'wrong beginner call' ) to decide to do a auto
**bun init**

and even make the index.ts like with a bun serve index.html ?

______________________
p.s. besides by bad english, i am not aggressive, never intended to have a 'tone' and only want help to promote / improve BUN
BUT
i not work with it, just look once a month on news, do a upgrade...
usually play my other hobby: micro-controller

**Comment by RiskyMH** [comment]
we should fix the root cause of this issue though (where it throws a confusing message for unregistered extensions).
```js
> bun app.css
error: File not found "app.css"
```

**Comment by zntchantchou** [comment]
@RiskyMH You are correct my global bun  version was outdated now with 1.3.1 the command "bun ./index.html" or even "bun index.html" runs as intended
